src/plugins/intel_gna/src/frontend/model_quantizer.hpp

```diff
@@ -141,7 +141,7 @@ class ModelQuantizer {
                     break;
                 }
 
-                prefix_function.emplace(i, k);
+                prefix_function.emplace(i, static_cast<int>(k));
```

Comment:
can prefix_function have negative values? if not maybe it would be good to change map parameters types to size_t/uint32_t?

Reply:
`prefix_function` can't have negative values, but `i` can, in fact `i >= 0` is a condition for searching an infinite loop; i've changed map parameter types to <int, size_t>